charts/fluent-bit/templates/servicemonitor.yaml

```diff
@@ -21,6 +22,14 @@ spec:
       {{- with .Values.serviceMonitor.scrapeTimeout }}
       scrapeTimeout: {{ . }}
       {{- end }}
+{{- if .Values.serviceMonitor.metricRelabelings }}
+      metricRelabelings:
+{{ tpl (toYaml .Values.serviceMonitor.metricRelabelings | indent 6) . }}
+{{- end }}
+{{- if .Values.serviceMonitor.relabelings }}
+      relabelings:
+{{ toYaml .Values.serviceMonitor.relabelings | indent 6 }}
```

Comment:
Could you use `nindent`.

Reply:
fixed as suggested

Reply:
This chart uses indented arrays, so could you increase the indent here and in the values?

Reply:
last commit has added also the indentation, I think is what you ask for...

Reply:
It should be indented to 8. The values should also show the array items indented.

Reply:
I think now I follow all yours suggestion.
I really appreciate your patience and guide.
Thanks a lot for all the comments.